### DISCUSSION TITLE
How to modify the layer of a line

### DISCUSSION THREAD

**Discussion by yingyue4008** [LibreDWG/libredwg#623] [Q&A] [answered]
After replacing the line layer, the lines have reached layer 0，

I don't know where the layer handle on the line is recorded，is dwg.object[i].tio.entity->layer  or dwg.object[i].tio.object->tio.LAYER->xref？

How do I replace the layer on the line？

**Comment by rurban** [comment, marked as answer]
It's dwg.object[i].tio.entity->layer, which is a handle pointing to the layer.

**Reply by yingyue4008** [reply]
> It's dwg.object[i].tio.entity->layer, which is a handle pointing to the layer.

Thank you very much，Can be set.
